pulpcore/cli/common/context.py

```diff
@@ -133,6 +147,49 @@ def __init__(
         self.timeout: int = timeout
         self.start_time: Optional[datetime.datetime] = None
 
+    def _patch_api_spec(self) -> None:
+        # A place for last minute fixes to the api_spec.
+        # WARNING: Operations are already indexed at this point.
+        api_spec = self.api.api_spec
+        if self.has_plugin(PluginRequirement("core", max="3.20.0")):
```

Comment:
Wonder if there is a way to put this in a separate file(s). It's possible we could end up with a bunch of small little spec changes we need to prevent bugs, and that would make this function become quite bloated.